Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=16.20.2-bullseye-slim
+ARG NODE_VERSION=22.14.0-bookworm
 
```

Comment:
At the moment, the image includes python, for the scripts in the `utapi` folder (c.f. https://scality.atlassian.net/browse/S3UTILS-172)

With the upgrade, a newer version of python will be used: so we need to check that the scripts still work (or find a way to remove them and drop python from the image, i.e. move forward with s3utils-172...)

Reply:
there is now `22.15.0-bookworm-slim`, should probably use it?